R/key-helpers.R

```diff
@@ -12,12 +12,13 @@
 #'   Variable names can be treated as if they were positions, so you
 #'   can use expressions like x:y to select ranges of variables.
 #'
-#'   The arguments in ... are automatically quoted and evaluated in a context where column names represent column positions.
+#'   The arguments in `...` are automatically quoted and evaluated in a context
+#'   where column names represent column positions.
 #'   They also support
 #'   unquoting and splicing.
-#'   See vignette("programming") for an introduction to these concepts.
```

Comment:
I don't see any vignette called "programming", so I'm assuming this is outdated?

Reply:
dplyr